cmd/vulcan-exposed-db/main.go

```diff
@@ -95,18 +92,14 @@ func exposedDatabases(target string, nmapReport *gonmap.NmapRun, databaseRegex *
 				"Version":  port.Service.Version,
 			}
 			gr.Rows = append(gr.Rows, networkResource)
-
-			e.WithFields(logrus.Fields{"resource": networkResource}).Debug("Resource added")
+			v := exposedVuln
+			v.Resources = []report.ResourcesGroup{gr}
+			v.Fingerprint = helpers.ComputeFingerprint(port.Service.Product, port.Service.Version)
+			v.Labels = []string{"issue", "db"}
+			vulns = append(vulns, v)
```

Comment:
Are we missing to define the `AffectedResource` field?

Reply:
Yes we are, I'll add it right away! Thanks for noticing.